rust-toolchain.toml

```diff
@@ -0,0 +1,4 @@
+[toolchain]
+channel = "1.70.0"
```

Comment:
The pipeline had an issue with the latest stable (`1.71.1`) so I added this to lock down the version. This was something on my todo list anyway.